src/app/game-state/activity.service.ts

```diff
@@ -619,27 +631,27 @@ export class ActivityService {
       level: 0,
       name: ['Attach Chains to the Island'],
       activityType: ActivityType.AttachChains,
-      description: ['Swim deep and attach one of your chains to the island.'],
-      consequenceDescription: ['Uses 1000 Stamina. Requires an unbreakable chain.'],
+      description: ['Swim deep and attach one of your chains to the island, then pull.'],
+      consequenceDescription: ['Uses 1000000 Stamina when carrying something REALLY heavy to try hauling something really REALLY heavy. You\'re going to need an Unbreakable Chain and a long time to rest.'],
```

Comment:
Is that the stamina cap? If it is, maybe raise the cap.

Reply:
I mean, is there some reason the cap needs raising? We literally have no use for huge huge stam other than me using it specifically for this.